src/manifest.c

```diff
@@ -574,29 +574,78 @@ static void set_untracked_manifest_files(struct manifest *manifest)
 	}
 }
 
-//NOTE: file==NULL when component=="MoM", else file->hash is needed
-int load_manifests(int current, int version, char *component, struct file *file, struct manifest **manifest)
+/* Loads the MoM (Manifest of Manifests) for VERSION.
+ * MOM manifest.
+ *
+ * Note that if the manifest fails to download, or if the manifest fails to be
+ * loaded into memory, this function will return NULL.
+ */
+struct manifest *load_mom(int version)
 {
+	struct manifest *manifest = NULL;
 	int ret = 0;
 
-	*manifest = manifest_from_file(version, component);
+	ret = retrieve_manifests(version, version, "MoM", NULL);
```

Comment:
Isn't this ALWAYS going to perform a curl_get_file on the MoM, regardless of whether or not it is already on the filesystem?